bin.js

```diff
@@ -421,6 +427,13 @@ function catchify (asyncFn) {
   }
 }
 
+function checkArgs (args,help,version) {
```

Comment:
style nits :-) .... no space between function name and opening (, and spaces separating the arg list

```suggestion
function checkArgs(args, help, version) {
```

Reply:
The space is actually required by Standard! :smile:

Reply:
Doh! Maybe eventually someday someone will convince node.js core to adopt standard 😂

Reply:
that sounds like a completely uncontroversial change that i'm sure would be adopted as soon as someone asks for it :upside_down_face: